README.md

```diff
@@ -161,14 +170,17 @@ See [C++ Programmer's Toolbox](#c-programmers-toolbox) for details.
 
 1.  **Parameters**
     *   Ordering: Input and then Output.
-    *   All **references** must be **const**, and this is the recommended
-        way to pass parameters.
-    *   For output parameters, pass pointers.
+    *   Either use const references or pointer for non-fundamental types. The stl types are not fundamental types.
```

Comment:
This is mainly for consistency, so that when the reader sees a pointer, he/she realizes that the function works by mutating this argument. Care to share the reasons for rewriting this rule?